src/previous.js

```diff
@@ -0,0 +1,18 @@
+import './styles.css';
+
+const taskList = document.querySelector('#task-list');
+
+const arr = [{
+  description: 'wash the dishes',
+  completed: 'false',
+  index: '0',
+}, {
+  description: 'complete to do list project',
+  completed: 'false',
+  index: '1',
+}];
+
+arr.forEach((items) => {
+  taskList.innerHTML += `
+    <div class="tasks"><input type="checkbox" name="first" value="first" id="first"><label class="tasks-text" for="first">${items.description}</label><span class="tasks-icon"><i class="fa fa-ellipsis-v" aria-hidden="true"></i></span></div>`;
+});
```

Comment:
- [ ] Please remove the previous hardcoded array. I understand that you are not using it but you code will look unprofessional if you have unused code there
![image](https://user-images.githubusercontent.com/62705947/229159867-7482e896-d02a-477a-825b-5d2e85ece13d.png)